parm/wflow/prep.yaml

```diff
@@ -6,33 +6,34 @@ default_task_prep: &default_task
   attrs:
     cycledefs: at_start
     maxtries: '2'
-  envars: &default_envars
-    GLOBAL_VAR_DEFNS_FP: '&GLOBAL_VAR_DEFNS_FP;'
-    USHdir: '&USHdir;'
-    PDY: !cycstr "@Y@m@d"
-    cyc: !cycstr "@H"
-    subcyc: !cycstr "@M"
-    LOGDIR: !cycstr "&LOGDIR;"
-    nprocs: '{{ parent.nnodes * parent.ppn }}'
   native: '{{ platform.SCHED_NATIVE_CMD }}'
-  nodes: '{{ nnodes }}:ppn={{ ppn }}'
-  nnodes: 1
-  ppn: 24
-  partition: '{% if platform.get("PARTITION_DEFAULT") %}&PARTITION_DEFAULT;{% else %}None{% endif %}'
```

Comment:
A lot of these default values are no longer needed after 2/3 of the below tasks are now running UW Drivers.